loggly/handlers.py

```diff
@@ -9,10 +9,12 @@
 session = FuturesSession()
 
 
-def bg_cb(sess, resp):
+
+def bg_cb(resp, *args, **kwargs):
```

Comment:
I prefer to rename it to response_hook because it is not background callback anymore.